src/content/app/genome-browser/state/track-config/trackConfigSlice.ts

```diff
@@ -21,43 +21,73 @@ import {
   PayloadAction,
   ThunkAction
 } from '@reduxjs/toolkit';
+import { RootState } from 'src/store';
 
 import browserStorageService from 'src/content/app/genome-browser/services/browserStorageService';
+import browserTrackConfigStorageService from '../../components/browser-track-config/services/browser-track-config-storage-service';
+import { BrowserTrackStates } from 'src/content/app/genome-browser/components/track-panel/trackPanelConfig';
 
 import { updateTrackStates } from 'src/content/app/genome-browser/state/browser-general/browserGeneralSlice';
 import { getBrowserTrackStates } from 'src/content/app/genome-browser/state/browser-general/browserGeneralSelectors';
 
-import { BrowserTrackStates } from 'src/content/app/genome-browser/components/track-panel/trackPanelConfig';
-import { RootState } from 'src/store';
+import { GenomeTrackCategory } from 'src/shared/state/genome/genomeTypes';
 
 export type CogList = {
   [key: string]: number;
 };
 
-export type TrackConfigState = Readonly<{
+export enum TrackType {
+  GENE = 'gene',
+  REGULAR = 'regular'
+}
+
+export const getTrackType = (trackId: string) => {
+  if (trackId.startsWith('gene')) {
+    return TrackType.GENE;
+  } else {
+    return TrackType.REGULAR;
+  }
+};
```

Comment:
While this will work for now, this is something we should discuss with the backend:
- How will we know on the frontend which settings panel to show for which track? Will the tracks we receive from the track api need to have a type field?
- What's the proper way for us to know the id of the focus track? It doesn't come from the track api. Is this something we just hard-code on the client? Will the id of the focus track change when a variation track is in focus? All this is something to discuss with Dan; possibly during the genome browser meeting tomorrow.

Reply:
Discussed with BE. We need to create JIRA tickets from the action points from the meeting